extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -77,12 +77,46 @@ private function getArticleDetails( $articleId ) {
 		$articleDetails = $this->sendRequest( 'ArticlesApi', 'getDetails', [ 'ids' => $articleId ] )
 			->getData()[ 'items' ][ $articleId ];
 
+		$description = $this->getArticleDescription( $articleId );
+
 		$articleDetails[ 'abstract' ] = htmlspecialchars( $articleDetails[ 'abstract' ] );
+		$articleDetails[ 'description' ] = htmlspecialchars( $description );
 
 		return $articleDetails;
 	}
 
 	/**
+	 * @desc Returns description for the article's meta tag.
+	 *
+	 * This is mostly copied from the ArticleMetaDescription extension.
+	 *
+	 * @param int $articleId
+	 * @param int $descLength
+	 * @return string
+	 * @throws WikiaException
+	 */
+	private function getArticleDescription( $articleId, $descLength = 100 ) {
+		$article = Article::newFromID( $articleId );
+		$title = $article->getTitle();
+		$sMessage = null;
+
+		if ( $title->isMainPage() ) {
+			// we're on Main Page, check MediaWiki:Description message
+			$sMessage = wfMessage( 'Description' )->text();
```

Comment:
It's copied from https://github.com/Wikia/app/blob/dev/extensions/wikia/ArticleMetaDescription/ArticleMetaDescription.php#L64

But from what I see here: https://github.com/Wikia/app/blob/dev/includes/GlobalFunctions.php#L1358
`wfMessage` always returns instance of `Message`. Should be safe.